### ISSUE TITLE
OG whisper word level timestamps support

### ISSUE THREAD

**Issue by Saccarab** [m-bain/whisperX#286]
is it possible to use OG whisper word-level timestamps and skip forced alignment?

**Comment by m-bain** [comment]
You will likely need v2 for that https://github.com/m-bain/whisperX/issues/232#issuecomment-1546460436

The batched inference does not currently support word level timestamps as of yet

**Comment by Saccarab** [comment]
would it be possible in theory to enable word_level timestamps through faster_whisper and patch them into the segments?

**Comment by stri8ed** [comment]
> would it be possible in theory to enable word_level timestamps through faster_whisper and patch them into the segments?

Yes. The timestamp tokens are being [filtered out during decoding](https://github.com/m-bain/whisperX/blob/78dcfaab51005aa703ee21375f81ed31bc248560/whisperx/asr.py#L69). You can remove the filtering, and then process them as needed.
